ui/client/components/graph/node-modal/editors/expression/EditableExpression.js

```diff
@@ -28,20 +33,22 @@ export default class EditableExpression extends React.Component {
     />
   }
 
-  toggleEditor = (_) => this.setState({
-    displayRawEditor: !this.state.displayRawEditor
-  })
-
-  switchable = (editorName, expressionObj) => {
-    switch (editorName) {
-      case Types.BOOL_EDITOR:
-        return true
-      case Types.RAW_EDITOR:
-        return editorType.switchableToBooleanEditor(expressionObj)
+  editorName = (fieldType, expressionObj, displayRawEditor) => {
```

Comment:
Let's just iterate on editors instead of switch? Only with some RawEditor special handling?

Reply:
Done